Writerside/topics/Matchmaking.md

```diff
@@ -1,2 +1,252 @@
+
+```json
+"Catena": {
+    "Matchmaker": {
+        "MatchmakingQueues": {
+            "up_to_3": {
+                "QueueName": "up_to_3",
+                "Teams": 0,
+                "PlayersPerTeam": 3,
+                "TicketExpirationSeconds": 180,
+                "ExpirationAction": "minimum"
+            }
+        }
+    },
+    "StatusExpirationMinutes": 15
+}
+```
+
+Since the expiration action must be supported by the strategy, no particular expiration action is guaranteed. However, all the built-in strategies in Catena support the following:
+
+| ExpirationAction | Description                                                                                                                                                                                                                                                                                           |
+|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| minimum          | Emits a match using the minimum number of expired tickets.<br/>This may be preferred so other players whose tickets have not expired still have a chance to be put in better fit or full matches later. It may lead to an overall higher match count which may increase server capacity requirements. |
+
+## Starting matchmaking
+
+Creating a ticket for matchmaking is done by submitting an `Entity`. Since entities can contain metadata and can be nested there are many different arrangements supported by the Catena matchmaker to accommodate structuring game data relevant to matchmaking.
+
+The entity metadata may be used to store game data such as character, map, or item selection for matches, players, or parties. Metadata passed to the matchmaker can be used during matchmaking and/or passed on to the clients or game servers in a match.
+
+> The Catena matchmaker requires that the top-level entity contain `queue_name` in the metadata, so it can assign the ticket to the correct queue. This may be sent by a client (not necessarily safe) or set/overridden within a [custom hook](#custom-hooks) based on ticket data (an example of this is contained in `QueueAssignmentMatchmakingHooks`).
+
+### Single player example
+
+In this example a player submits a ticket for themselves to enter the 1v1 queue.
+
+The additional `map` metadata could be used by a [custom strategy](#custom-strategies) to ensure this player is only matched with another player choosing the same map. It could also be added to the match data by a [custom hook](#custom-hooks), either to select a server already running that map or to send the map choice to the server to load it when it receives the match.
+
+```json
```

Comment:
It's worth showing _where_ the user would submit this, ie the StartMatchmaking1v1 request, you should be able to include the most up-to-date version by including a reference to the OpenAPI call.

Reply:
I think we only need to do this once rather than repeating ourselves for each.